### DISCUSSION TITLE
How to get only the Preprocessed OpenPose preview via API http request code?

### DISCUSSION THREAD

**Discussion by etekinalp** [Mikubill/sd-webui-controlnet#2244] [Q&A] [answered]
Hello everybody,

I am trying to figure out how I can get the openpose preprocessed result only via http request. In the UI we would select the openpose preprocessor and then hit the fire icon between preprocessor and model. I need exactly that but via code.

I was not able to find an answer in the forum nor the /docs 

Any ideas?

Thank you very much!

**Comment by huchenlei** [comment, marked as answer]
See https://github.com/Mikubill/sd-webui-controlnet/wiki/API#route-post-controlnetdetect
